applications/asset_tracker_v2/src/cloud/nrf_cloud_integration.c

```diff
@@ -82,9 +82,6 @@ static int fmfu_and_modem_init(const struct device *dev)
 		}
 	}
 
-	/* Ignore the result, it will be checked later */
-	(void)nrf_modem_lib_init(NORMAL_MODE);
```

Comment:
cc @jayteemo.

Reply:
looks good; this is no longer needed since `nrf_modem_lib_init` is now called in main().